rspotify-model/src/idtypes.rs

```diff
@@ -210,6 +210,7 @@ macro_rules! define_idtypes {
                 /// Only returns `true` in case the given string is valid
                 /// according to that specific ID (e.g., some may require
                 /// alphanumeric characters only).
+                #[must_use]
```

Comment:
Just out of curiosity, these functions are all `must_use` ?

Reply:
What do you mean? Yes, I marked all of them with must_use because clippy told me to, and it makes sense.